test/functions.php

```diff
@@ -6,5 +6,11 @@
 
 class functions extends TestCase
 {
-    public function testCollectionDbConnection
-}
\ No newline at end of file
+
+    public function testDisplayData()
+    {
+        $cars = [['make'=>"vw"],['make'=>'bmw']];
+        $result = displayData($cars);
+        $this->assertTrue(is_string($result));
```

Comment:
maybe test the contents of the result as well as the data type

Reply:
added the below assertion